### ISSUE TITLE
Sign in with Microsoft Account

### ISSUE THREAD

**Issue by ewgdg** [PlayFab/vscode-playfab-explorer#81]
There is no way to sign in my Microsoft Account for this extension.

**Comment by ewgdg** [comment]
Yes, I sign in to playfab.com by clicking the "Sign in with Microsoft" button on the page. I am not sure if it is Azure Active Directory flow. 
I realized that this extension is not very popular, so I guess you can close it if nobody cares.

**Comment by MGudgin** [comment]
Yes, that is the Active Directory flow. We've had another request to support this. I will try to get to it at some point soon.

**Comment by jkarkkainen-vaki** [comment]
I could use this functionality too, as I use a Microsoft account to login to PlayFab.

**Comment by jaced886** [comment]
bra
